source-stripe/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim
```

Comment:
There are leftover files in the root directory from the import that we don't need anymore.